tests/unit/backend/recipe/algorithm/test_SmoothDataExcludingPeaksAlgo.py

```diff
@@ -41,53 +43,6 @@ def test_unbag_groceries(self):
         assert algo.getPropertyValue("InputWorkspace") == testWS.name()
         assert algo.getPropertyValue("Outputworkspace") == "test_out_ws"
 
-    def test_validate_success_with_peaks(self):
-        testWS = CreateSingleValuedWorkspace()
-        algo = Algo()
-        algo.initialize()
-        algo.setProperty("InputWorkspace", testWS)
-        algo.setPropertyValue("OutputWorkspace", "test_out_ws")
-        algo.setProperty("DetectorPeaks", "[{}]")
-        algo.setProperty("SmoothingParameter", 0.0)
-        errors = algo.validateInputs()
-        assert errors == {}
-
-    def test_validate_success_with_ingredients(self):
-        testWS = CreateSingleValuedWorkspace()
-        ingredientString = Resource.read("inputs/predict_peaks/input_fake_ingredients.json")
-        algo = Algo()
-        algo.initialize()
-        algo.setProperty("InputWorkspace", testWS)
-        algo.setPropertyValue("OutputWorkspace", "test_out_ws")
-        algo.setPropertyValue("DetectorPeakIngredients", ingredientString)
-        algo.setProperty("SmoothingParameter", 0.0)
-        errors = algo.validateInputs()
-        assert errors == {}
-
-    def test_validate_fail_no_peaks(self):
-        # check that it will fail if it is not given a way to find peaks
-        testWS = CreateSingleValuedWorkspace()
-        algo = Algo()
-        algo.initialize()
-        algo.setProperty("InputWorkspace", testWS)
-        algo.setPropertyValue("OutputWorkspace", "test_out_ws")
-        algo.setProperty("SmoothingParameter", 0.0)
-        errors = algo.validateInputs()
-        assert errors.get("DetectorPeaks") is not None
-        assert errors.get("DetectorPeakIngredients") is not None
-
-    def test_validate_fail_no_smoothing(self):
-        # check that it will fail if it is not given a way to find peaks
-        testWS = CreateSingleValuedWorkspace()
-        algo = Algo()
-        algo.initialize()
-        algo.setProperty("InputWorkspace", testWS)
-        algo.setPropertyValue("OutputWorkspace", "test_out_ws")
-        algo.setProperty("DetectorPeaks", "[{}]")
-        errors = algo.validateInputs()
-        assert errors.get("DetectorPeakIngredients") is not None
-        assert errors.get("SmoothingParameter")
-
```

Comment:
Note: these were all tests of the validation method, which used to check whether one of DetectorPeaks or DetectorPeakIngredients was set, but was deleted.